ru/paid_features.md

```diff
@@ -0,0 +1,48 @@
+**[Оглавление](index)	>	Платные возможности**
+
+Платные возможности доступны только когда приложение установлено [через Google Play](https://play.google.com/store/apps/details?id=com.emanuelef.remote_capture). Если Вы пользователь F-Droid и у вас есть какие-либо предложения про возможность приема платежей через сборки для F-Droid, пожалуйста [свяжитесь со мной по e-mail](mailto:black.silver@hotmail.it?subject=PCAPdroid)
+
+## 5.1 Обнаружение вредоносных соединений
+
+**Дисклеймер**: *Функционал обнаружения вредоносных соединений в PCAPdroid не является комплесным решением для обеспечения безопасности устройства. Автор не предоставляет никаких гарантий относительно возможностей обнаружения вредоносных соединений или их точности в PCAPdroid, и не несет ответственности за любой прямой или косвенный ущерб причиненный использованием данной возможности.*
+
+Функция обнаружения вредоносов позволяет PCAPdroid обнаруживать соединения с вредоносными хостами благодаря сторонним черным спискам. Обнаружение активно только во время захвата трафика.
+
+Сегодня перед нашими устройствами стоит множество угроз: фишинг, онлайн мошенничество, вымогатели и слежка, а так же множество других. Когда заходит вопрос о безопасности, осторожности никогда не бывает достаточно и ни одно решение не покроет все возможные случаи. Возможность обнаружения вредоносов в PCAPdroid может помочь обнаруживать вредоносные соединения по мере появления таковых, тем самым обращая внимание пользователя на возможную угрозу.
+
+Некоторые случаи в которых эта возможность применима:
+
+- Пользователь посещает известный опасный сайт (например фишинговый или мошеннический)
+- Пользователь устанавливает некое приложение или дополнение (которое оказывается шпионит) которое соединяется с опасным доменом или IP адресом
+- На устройство любым образом попадает вредонос (например шпион, вымогатель или ботнет), который соединяется с опасным доменом или IP адресом
+
+Черные списки используемые в PCAPdroid содержат списки доменов и IP адресов с плохой репутацией. Они включают сканнеры, брутфорсеров и известные источники распространения вредоносного ПО. Черные списки в основном содержат статические правила, основанные на последних данных о заражениях, а так же некоторые динамические правила, которые сгенерированы автоматически, обычно благодаря [АСООС](https://ru.wikipedia.org/wiki/Client_honeypot)(англ. [Client Honeypots](https://en.wikipedia.org/wiki/Client_honeypot)). PCAPdroid обновляет списки раз в сутки, чтобы иметь возможность обнаруживать актуальные вредоносные соединения.
```

Comment:
What's ACOOC? Please replace the reference to Client Honeypots with a generic Honeypots (link: https://ru.wikipedia.org/wiki/Honeypot), sorry for providing the wrong link in tg

Reply:
fixed in recent commit